prow/cmd/deck/job_history.go

```diff
@@ -297,8 +297,9 @@ func getBuildData(bucket storageBucket, dir string) (buildData, error) {
 		return b, fmt.Errorf("failed to read started.json: %v", err)
 	}
 	b.Started = time.Unix(started.Timestamp, 0)
-	commitHash, err := getPullCommitHash(started.Pull)
-	if err == nil {
+	if started.Revision != "" {
```

Comment:
Maybe worth check both started and finished? We actually write revision in finished.json - (https://github.com/kubernetes/test-infra/blob/master/prow/sidecar/run.go#L189-L193) but technically you can also write it in started.

Reply:
done